packages/wearable_rotary/lib/wearable_rotary.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/services.dart';
+
+const String _channelName = 'flutter.wearable_rotary.channel';
+
+const EventChannel _channel = EventChannel(_channelName);
+
+Stream<RotaryEvent> get rotaryEvent {
+  _rotaryEvent ??= _channel
```

Comment:
```suggestion
Stream<RotaryEvent> get rotaryEvents {
  _rotaryEvents ??= _channel
```
The plural form is common for event streams.